dependencies.sh

```diff
@@ -37,6 +37,19 @@ dnf install -y \
 
 systemctl enable osbuild-composer.socket cockpit.socket --now
 
+# Setup libvirt modular
+systemctl stop libvirtd.service
+systemctl stop libvirtd{,-ro,-admin,-tcp,-tls}.socket
+
+systemctl disable libvirtd.service
+systemctl disable libvirtd{,-ro,-admin,-tcp,-tls}.socket
+
+for drv in qemu interface network nodedev nwfilter secret storage; do
+  systemctl enable virt${drv}d.service
+  systemctl enable virt${drv}d{,-ro,-admin}.socket
+  systemctl start virt${drv}d{,-ro,-admin}.socket
+done
```

Comment:
should we do this in python instead?

Reply:
We do, but that's during the initial installation of all dependencies

Reply:
why would we do some of the stuff in sh and some in python?

Reply:
It's safer to switch to modular during the first installation, the README suggested to enable libvirtd before everything

Reply:
The intent in this project was to avoid `sh`. The only reason I initially added the `.sh` file was to bootstrap everything. If you do a `git blame` on the file, it seems that this file has grown quite a bit and no lines remain from the original code. I'd like to avoid adding logic to this file. 

Safer in what sense? Can't we have a function at the start of cda to enable this? What if you don't even need VMs for a specific yaml. You would need to read the yaml file in bash to determine if you need to enable the systemd. I'd like to explore how the solution would look like in python. Can you propose a change fully in python instead?

Reply:
Well that seems to be happening even if I remove the shell code:

`main_deploy()` -> `main_deploy_openshift()` -> `VirBridge.configure()` -> `Libvirt.configure()` 

so in that case I'll remove the shell code.